cmd/meta/main.go

```diff
@@ -289,6 +294,81 @@ func (m *Main) RunMusicBrainzIndex(args Args) error {
 	return indexer.IndexArtists(ctx, stream)
 }
 
+func (m *Main) RunCwr(args Args) error {
+	switch {
+	case args.Bool("convert"):
+		return m.RunCwrConvert(args)
+	case args.Bool("index"):
+		return m.RunCwrIndex(args)
+	default:
+		return errors.New("unknown cwr command")
+	}
+}
+
+func (m *Main) RunCwrConvert(args Args) error {
+
+	// stream the CIDs to stdout
+	stream := make(chan *cid.Cid)
+	go func() {
+		for cid := range stream {
+			fmt.Fprintln(os.Stdout, cid.String())
+		}
+	}()
+
+	// shutdown gracefully on SIGINT or SIGTERM
+	ctx, cancel := context.WithCancel(context.Background())
+	go func() {
+		defer cancel()
+		ch := make(chan os.Signal, 1)
+		signal.Notify(ch, syscall.SIGINT, syscall.SIGTERM)
+		<-ch
+		log.Info("received signal, exiting...")
+	}()
+
+	return cwr.NewConverter(args.String("<file>"), m.store).ConvertRegisteredWork(ctx, stream)
```

Comment:
The file should be passed to `ConvertRegisteredWork` so that you can re-use a `Converter` to convert multiple files, and we should open the file here and pass an `io.Reader` rather than a `string`.